src/Microsoft.Identity.Web.DownstreamRestApi/DownstreamRestApi.HttpMethods.cs

```diff
@@ -0,0 +1,351 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Abstractions;
+
+namespace Microsoft.Identity.Web
+{
+    /// <inheritdoc/>
+    internal partial class DownstreamRestApi : IDownstreamRestApi
+    {
+
+        /// <inheritdoc/>
+        public async Task<TOutput?> GetForUserAsync<TOutput>(
+            string? serviceName,
+            Action<DownstreamRestApiOptionsReadOnlyHttpMethod>? downstreamRestApiOptionsOverride = null,
+            ClaimsPrincipal? user = null,
+            CancellationToken cancellationToken = default)
+            where TOutput : class
+        {
+            DownstreamRestApiOptions effectiveOptions = MergeOptions(serviceName, downstreamRestApiOptionsOverride, HttpMethod.Get);
+            HttpResponseMessage response = await CallRestApiInternalAsync(serviceName, effectiveOptions, false,
+                                                                          null, user, cancellationToken).ConfigureAwait(false);
+
+
+            return await DeserializeOutput<TOutput>(response, effectiveOptions).ConfigureAwait(false);
+        }
+
+        /// <inheritdoc/>
+        public async Task<TOutput?> GetForUserAsync<TInput, TOutput>(
+            string? serviceName,
+            TInput input,
+            Action<DownstreamRestApiOptionsReadOnlyHttpMethod>? downstreamRestApiOptionsOverride = null,
+            ClaimsPrincipal? user = null,
+            CancellationToken cancellationToken = default)
+            where TOutput : class
+        {
+            DownstreamRestApiOptions effectiveOptions = MergeOptions(serviceName, downstreamRestApiOptionsOverride, HttpMethod.Get);
+            HttpContent? effectiveInput = SerializeInput(input, effectiveOptions);
+            HttpResponseMessage response = await CallRestApiInternalAsync(serviceName, effectiveOptions, false,
+                                                                          effectiveInput, user, cancellationToken).ConfigureAwait(false);
```

Comment:
nit: either put all the parameters on one line or move them all in-line on a separate line.

Reply:
Done